### ISSUE TITLE
Maven Repo down

### ISSUE THREAD

**Issue by CrazyDev05** [DaRacci/slimjar#23]
The maven repository https://repo.racci.dev/ doesn respond anymore

**Comment by CrazyDev05** [comment]
Hey @DaRacci as it seems like you lost interest in maintaining the project I decided to nake my own fork which is released on maven central to mitigate this issue in the future
https://github.com/CrazyDev05/slimjar

**Comment by DaRacci** [comment]
Yeah sorry, my repo web server went down in flames a while back and since I no longer do any java related coding I didn't bother setting it back up.

Thanks for forking to maintain, I'm going to archive this repository but before I do would you like me to add a note to the readme that redirects to your fork if you are up for maintaining it?

**Comment by CrazyDev05** [comment]
sure that would be nice
